src/app/static/src/app/store/root/actions.ts

```diff
@@ -72,5 +90,21 @@ export const actions: ActionTree<RootState, RootState> & RootActions = {
 
         await Promise.all(actions);
         commit({type: RootMutation.SetUpdatingLanguage, payload: false});
+    },
+
+    generateErrorReport({state, getters}, payload) {
+        // TODO: instead of returning this object, POST it to the server
+        return {
+            email: payload.email || state.currentUser,
+            country: state.baseline.country,
+            project: state.projects.currentProject?.name,
+            browserAgent: navigator.userAgent,
+            timeStamp: new Date().toISOString(),
+            jobId: state.modelRun.modelRunId,
+            description: payload.description,
+            section: payload.section,
+            stepsToReproduce: payload.stepsToReproduce,
+            errors: getters.errors
+        }
```

Comment:
It would be nice to wrap this object in a type. JobId and errorMessage are type of array.

```
export interface ErrorReportMetadata {
    email: string,
    country: string,
    projectName: string,
    section: string,
    errors: ErrorReportMessage[],
    "description": string,
    "stepsToReproduce": string,
    "browserAgent": string
}
```

```
interface ErrorReportMessage {
    jobId: string,
    errorMessage: string
}
```